addresses/mainnet.json

```diff
@@ -1,394 +1,418 @@
+          "fee": "0.1",
+          "timeStretch": 18.5
+        },
+        "ytPool": {
+          "address": "0x2D6e3515C8b47192Ca3913770fa741d3C4Dac354",
+          "poolId": "0x2d6e3515c8b47192ca3913770fa741d3c4dac35400020000000000000000007b",
+          "fee": "0.003"
+        },
+        "weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+        "convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+      },
+      {
+        "expiration": 1643382476,
+        "address": "0x8a2228705ec979961F0e16df311dEbcf097A2766",
+        "trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+        "ptPool": {
+          "address": "0x10a2F8bd81Ee2898D7eD18fb8f114034a549FA59",
+          "poolId": "0x10a2f8bd81ee2898d7ed18fb8f114034a549fa59000200000000000000000090",
+          "fee": "0.1",
+          "timeStretch": 24.7
+        },
+        "ytPool": {
+          "address": "0x9e030b67a8384cbba09D5927533Aa98010C87d91",
+          "poolId": "0x9e030b67a8384cbba09d5927533aa98010c87d9100020000000000000000008f",
+          "fee": "0.003"
+        },
+        "weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+        "convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+      },
+      {
+        "expiration": 1639727861,
+        "address": "0x76a34D72b9CF97d972fB0e390eB053A37F211c74",
+        "trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+        "ytPool": {
+          "address": "0x7C9cF12d783821d5C63d8E9427aF5C44bAd92445",
+          "poolId": "0x7c9cf12d783821d5c63d8e9427af5c44bad924450002000000000000000000b4",
+          "fee": "0.003"
+        },
+        "ptPool": {
+          "address": "0x90ca5cef5b29342b229fb8ae2db5d8f4f894d652",
+          "poolId": "0x90ca5cef5b29342b229fb8ae2db5d8f4f894d6520002000000000000000000b5",
+          "fee": "0.1",
+          "timeStretch": 24.66
+        },
+        "weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+        "convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+      },
+      {
+        "expiration": 1651253068,
+        "address": "0x52C9886d5D87B0f06EbACBEff750B5Ffad5d17d9",
+        "trancheFactory": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
+        "ytPool": {
+          "address": "0x7173b184525feAD2fFbde5FBe6FCB65Ea8246eE7",
+          "poolId": "0x7173b184525fead2ffbde5fbe6fcb65ea8246ee70002000000000000000000c7",
+          "fee": "0.003"
+        },
+        "ptPool": {
+          "address": "0x7edde0cb05ed19e03a9a47cd5e53fc57fde1c80c",
+          "poolId": "0x7edde0cb05ed19e03a9a47cd5e53fc57fde1c80c0002000000000000000000c8",
+          "fee": "0.1",
+          "timeStretch": 24.66
+        },
+        "weightedPoolFactory": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+        "convergentCurvePoolFactory": "0xb7561f547F3207eDb42A6AfA42170Cd47ADD17BD"
+      }
```

Comment:
My editor removed the tabs in this file so this is just to clarify the changes

Reply:
why did you need to edit it?  did the deployment scripts not modify this for you?  Normally i prefer spaces to tabs, but i think its better to not have this huge diff.

Reply:
Do we not have to manually construct this object? We also had to introduce the "yearn4"  which you can see in the wrapped positions and vaults fields. I've a preference for spaces as tabs make it very wide but can undo if necessary

Reply:
hmm...my understanding was that if you entered yearnv4 for type it would have added it automatically.

if we are going to switch to spaces....i would prefer that be in a sep PR.  there are many times where i need to go back to one of these PRs to see what addresses were added and the diff is useful

Reply:
that's crazy that it was using tabs in the first place! who allowed this?!?! 😆.  I also prefer this to be in a separate PR but I don't think its a huge deal, I'm fine to land as is.

Reply:
Same question as @jrhea though, we shouldn't have to hand edit this file ever.  What's the current process to generate this file?  I'm a little behind on elf-deploy processes.

Reply:
Sarah can probably speak more to this then I but usually all going well the outputs of a few scripts will give us the necessary data that can be readily copied over. There are cases where a script might barf but the tx succeed correctly that might require investigative work to get the correct info and that might be copied over. I think this file could be auto generated if we run across the event log from the different factory contracts

Reply:
I think the generating of the file should be separated from the deployment transactions as they are deemed to fail easily